src/hooks/useWidth.js

```diff
@@ -5,7 +5,7 @@ export default function useWidth() {
 
   useEffect(() => {
```

Comment:
why do you need a hook for resize ?
add comments for methods

Reply:
I want to calculate the width so that's why i am using custom hook.